packages/trackerless-network/src/logic/protocol-integration/stream-message/GroupKeyRequestTranslator.ts

```diff
@@ -1,25 +1,26 @@
 import { GroupKeyRequest as OldGroupKeyRequest } from '@streamr/protocol'
 import { GroupKeyRequest } from '../../../proto/packages/trackerless-network/protos/NetworkRpc'
 import { EthereumAddress } from '@streamr/utils'
+import { toBinary, toUTF8 } from '../../utils'
 
 // eslint-disable-next-line @typescript-eslint/no-extraneous-class
 export class GroupKeyRequestTranslator {
 
     static toProtobuf(msg: OldGroupKeyRequest): GroupKeyRequest {
         const translated: GroupKeyRequest = {
-            recipient: msg.recipient,
+            recipient: toBinary(msg.recipient),
             requestId: msg.requestId,
-            rsaPublicKey: msg.rsaPublicKey,
+            rsaPublicKey: toBinary(msg.rsaPublicKey),
             groupKeyIds: msg.groupKeyIds
         }
         return translated
     }
 
     static toClientProtocol(msg: GroupKeyRequest): OldGroupKeyRequest {
         const translated = new OldGroupKeyRequest({
-            recipient: msg.recipient as EthereumAddress,
+            recipient: toUTF8(msg.recipient) as EthereumAddress,
```

Comment:
You need to use `toEthereumAddress` to ensure that the resulting `EthereumAddress` is lower-cased.